node/test/max-call-overhead.js

```diff
@@ -81,16 +81,23 @@ allocCluster.test('request() with large arg1', {
         serviceName: 'server'
     });
 
+    var arg1 = '';
+    for (var i = 0; i < 16 * 1024 + 1; i++) {
+        arg1 += 'a';
+    }
+
     subTwo.waitForIdentified({
         host: one.hostPort
     }, function onIdentified(err) {
         assert.ifError(err);
 
-        var arg1 = '';
-        for (var i = 0; i < 16 * 1024 + 1; i++) {
-            arg1 += 'a';
-        }
+        // TODO: O(pinions) -- is this really what we want?
```

Comment:
O(bservation): There will be cases of computed arg1, possibly including some HTTP translation layer, in which case it should be an operational error, not a hard stop and die.